index.bs

```diff
@@ -2304,17 +2310,17 @@ the {{ReadableStream}}'s public API.
  <dfn abstract-op lt="ReadableStreamClose"
  id="readable-stream-close">ReadableStreamClose(|stream|)</dfn> performs the following steps:
 
- 1. Assert: |stream|.\[[state]] is "`readable`".
- 1. Set |stream|.\[[state]] to "`closed`".
- 1. Let |reader| be |stream|.\[[reader]].
+ 1. Assert: |stream|.[=ReadableStream/[[state]]=] is "`readable`".
+ 1. Set |stream|.[=ReadableStream/[[state]]=] to "`closed`".
+ 1. Let |reader| be |stream|.[=ReadableStream/[[reader]]=].
  1. If |reader| is undefined, return.
  1. If |reader| [=implements=] {{ReadableStreamDefaultReader}},
-  1. [=list/For each=] |readRequest| of |reader|.\[[readRequests]],
+  1. [=list/For each=] |readRequest| of |reader|.[=ReadableStreamDefaultReader/[[readRequests]]=],
    1. Perform |readRequest|'s [=read request/close steps=].
-  1. Set |reader|.\[[readRequests]] to an empty [=list=].
- 1. [=Resolve=] |reader|.\[[closedPromise]] with undefined.
+  1. Set |reader|.[=ReadableStreamDefaultReader/[[readRequests]]=] to an empty [=list=].
+ 1. [=Resolve=] |reader|.[=ReadableStreamGenericReader/[[closedPromise]]=] with undefined.
 
- <p class="note">The case where |stream|.\[[state]] is "`closed`", but |stream|.\[[closeRequested]]
+ <p class="note">The case where |stream|.[=ReadableStream/[[state]]=] is "`closed`", but |stream|.[=ReadableStream/[[controller]]=].\[[closeRequested]]
```

Comment:
This is the only place where we refer to `[[closeRequested]]` without a specific type of controller. I think it's a bit overkill to introduce an interface mixin *just* for this non-normative note, so I'm tempted to only link to the internal slot on the default controller, i.e.:
```suggestion
 <p class="note">The case where |stream|.[=ReadableStream/[[state]]=] is "`closed`", but |stream|.[=ReadableStream/[[controller]]=].[=ReadableStreamDefaultController/[[closeRequested]]=]
```

Thoughts?

Reply:
That seems reasonable, or leaving it un-linked.

This note seems a little confusing though... the algorithm doesn't mention closeRequested, and it seems kind of redundant with https://streams.spec.whatwg.org/#readable-stream-default-controller-can-close-or-enqueue. So maybe it should just be deleted?

Reply:
Indeed, it seems redundant. I'll delete it. 🙂